deny.toml

```diff
@@ -17,12 +17,6 @@ default = "deny"
 # [possible values: any SPDX 3.11 short identifier (+ optional exception)].
 allow = [
     "BSD-2-Clause",
-    "LicenseRef-krator",
-    "LicenseRef-krator-derive",
-    "LicenseRef-ring",
-    "LicenseRef-webpki",
-    "LicenseRef-webpki-roots",
-    "MPL-2.0",
```

Comment:
I'm curious on why these licenses are being removed from the allow list (granted that I have failed to find some, like `LicenseRef-krator`, in the [SPDX list](https://spdx.org/licenses/)).

Reply:
I removed them because `cargo deny` was showing warning messages about these licenses not being used anymore. I guess they are there because the `deny.toml` was initially copied from the one of krustlet, which had way more dependencies.

However, I had to restore the old file because some licenses were actually being used when building on windows